deepfence_agent/artifacts.json

```diff
@@ -0,0 +1,58 @@
+{
+    "agentAuth": {
+        "source": "tools/apache/deepfence/df-utils/agent_auth/agentAuth",
+        "repository": ".",
+        "destination": "/usr/local/bin/agentAuth",
+        "service": null,
+        "lockfile": null
+    },
+    "getCloudInstanceId": {
+        "source": "tools/apache/deepfence/df-utils/get_cloud_instance_id/getCloudInstanceId",
+        "repository": ".",
+        "destination": "/usr/local/bin/getCloudInstanceId",
+        "service": null,
+        "lockfile": null
+    },
+    "deepfence-discovery": {
+        "source": "tools/apache/scope/docker/deepfence_exe",
+        "repository": ".",
+        "destination": "/usr/local/discovery/deepfence-discovery",
+        "service": "discovery",
+        "lockfile": null
+    },
+    "deepfence_compliance_check": {
+        "source": "tools/apache/compliance_check/deepfence_compliance_check",
```

Comment:
Might be wrong but `compliance` is also a scan so we should probably use a `lockfile` there as well?
Not sure what the diff is between `compliance` & `deepfence_compliance_check`